yt/yt/server/controller_agent/config.cpp

```diff
@@ -698,6 +698,10 @@ void TDockerRegistryConfig::Register(TRegistrar registrar)
 {
     registrar.Parameter("internal_registry_address", &TThis::InternalRegistryAddress)
         .Default();
+    registrar.Parameter("forward_yt_tokens_to_internal_registry", &TThis::ForwardYtTokensToInternalRegistry)
```

Comment:
`use_temporary_tokens_for_internal_registry`?

Reply:
this code known nothing about temporary or not this token is
it forwards any YT_TOKEN to internal registry

Reply:
so, `use_yt_token_for_internal_registry`